web/modules/custom/routing/routing.routing.yml

```diff
@@ -0,0 +1,8 @@
+route.routing:
+  path: /route
+  defaults:
+    _controller: Drupal\routing\Controller\RoutingController::myRoute
+    _title: Custom Route
+  requirements:
+    _permission: 'routing permission'
```

Comment:
who has this permission?
I cannot see the configuration files which would help me know who is having this permissions.

Reply:
Sir this ticket (FT2023-339( was only about knowing about permissions and generating permissions. Restricting users on the basis of the permissions they are having is a part of ticket FT2023-342, so i haven't implemented it in this one

I've implemented it here 
https://github.com/sandipta18/Drupal-Backend/pull/5

Reply:
You gave the permissions to some users roles right?
So that becomes part of the configuration, which I cannot see in your PR.

Reply:
Ok i will add the configurations

Reply:
Added the configurations please check.
I have one question, permissions assigned to a role only be checked inside user.role.rolename.yml files right? Like if we want to check it for admin we have to check user.role.administrator.yml file. Or are there any other options for viewing permissions assigned to a role ?

@vivekpan27

Reply:
Check if from the interface.